### PR TITLE
Update orbitalwise analysis

### LINKED ISSUE #258: Orbital contributions should be summed by orbital type
We should report on s-p interactions instead of s-px, s-py, s-pz interactions and so on as the individual contributions don't have a real meaning unless you rotate the orbitals beforehand.

### PR DESCRIPTION
**Pull request by naik-aakash**
Closes #258

# Changes

1. Orbital contributions are summed now (Eg: 2s- 3p orbital contribution consists of 2s-3px, 2s-3py, 2s-3py)
2. Add some private utility methods in Analysis class to achieve this

# Todo

- [x] Update tests and associated reference files
- [x] Update tutorial orbital resolved analysis plot html files and manual plotting section 
- [x] Update doc-strings where needed Eg:- [Line 677 of analyze](https://github.com/naik-aakash/LobsterPy/blob/0ec2265ff4a73fd8b3bf807d17d08726f95deaf2/lobsterpy/cohp/analyze.py#L667)

### CODE REVIEW

File: lobsterpy/cohp/analyze.py [hunk 1]
```diff
@@ -603,7 +614,7 @@ def _get_bond_resolved_data_stats(self, orb_resolved_bond_data: dict):
         Returns:
             dict with orbital data stats the site for relevant orbitals, e.g.
             {'orbital_summary_stats': {'max_bonding_contribution': {'2s-3s': 0.68},
-            'max_antibonding_contribution': {'2s-2pz': 0.36}}}
+            'max_antibonding_contribution': {'2s-2p': 0.36}}}
```
Comment (JaGeo) [inline lobsterpy/cohp/analyze.py:617]:
```suggestion
            'max_antibonding_contribution': {'2s-2p': 0.36}}}

```

Why does this stay the same depite "sum"?

Comment (naik-aakash) [reply lobsterpy/cohp/analyze.py:617]: I did not put exact values there. This is just the random number in doc-strings to indicate what to expect 😅

Comment (JaGeo) [reply lobsterpy/cohp/analyze.py:617]: Ah. Doc string. Sorry!

Comment (naik-aakash) [reply lobsterpy/cohp/analyze.py:617]: No worries. But I will update it to a reasonable number to avoid confusion

**Comment by naik-aakash** [comment]
## Updated condensed bonding analysis output for a single site

image

## Description text

image

## Updated output of `Analysis.get_site_orbital_resolved_labels()` method 
This could be used for plotting sub-orbital data if users have aligned the orbitals in their LOBSTER calculations. Also, at this point, the output from here is used to generate the automatic interactive plots.

image

## Updated automatic interactive plot

image

**Comment by JaGeo** [comment]
Could you test some more complex examples including d and f orbitals? Thx! @naik-aakash

**Review by JaGeo — APPROVED** [review]

**Comment by JaGeo** [comment]
@naik-aakash Should I directly release another version?

**Comment by naik-aakash** [comment]
Hi @JaGeo , wait I need to raise another PR with the kpoints note update in the docs

**Pull request merged**